foursq.js

```diff
@@ -0,0 +1,51 @@
+class Place {
+  constructor(name, cat, lat, lng, url, hereNow) {
+    this.name = name;
+    this.cat = cat;
+    this.lat = lat;
+    this.lng = lng;
+    this.url = url;
+    this.hereNow = hereNow;
+  }
+}
+
+// FourSquare API
+function fourSquareFunction(search) {
+  var places = [];
+
+  var radius = 8000; // meters
```

Comment:
Was reading the API doc for FourSquare and they have something called "intents" and then something called "browse" which will search a broader area. We may want to use this in our url instead of radius. Nothing to change here, more just an FYI

Reply:
Will look into 'intents'. Thanks for the info